### ISSUE TITLE
[Bug] Notification Bug 

### ISSUE THREAD

**Issue by girokarambole1453** [WFCD/navis#330]
### Describe the bug

I dont know if here is the correct place to report a bug that occurs in the Navis app. So I am sorry if its not. I have just installed the Navis and I got 4 notification saying that Baro has arrived. All 4 notifications are same. And I am wondering why I got that 4 times. Should I enter the app to make it stop sending notification? or is it a bug or something? (I have got another notification while writing this report xdd.) I am laughing because I loved the app and wanted it to get fixed. I am not kidding or anything else please dont get me wrong.

### How to Reproduce

1. Navis installed from Play Store
2.Navis opened, enable Baro notifications
3.Navis said Baro has arrived for 4 times
...
I dont know what I should write here...

### Expected Behavior

To notification to appears for supposed time.

### What platform are you using Navis on

Android

### Platform Version

Android 11, OneUI 3.1

### Cephalon Navis version

2.10.9

### Additional Context

I am new to the github sorry If anything wrong done by me :(

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by SlayerOrnstein** [comment]
It's a bug from when I tried to fix it Friday. Looking at it now

**Comment by girokarambole1453** [comment]
Thank you man, have a nice day!

**Comment by SlayerOrnstein** [comment]
fix should be pushed, since the notification script is stateless you'll get one notification for anything currently happening and then should go back to normal

**Comment by girokarambole1453** [comment]
I havent got any more Baro arrived notification 😄😄. Thank you for the fix man. Also I loved your app. Keep going, have a nice day!❤❤❤ (I guess I should close this report since its been fixed :)

**Comment by SlayerOrnstein** [comment]
Yeah I saw, it's something with the sentient outpost not being there. Since it's stateless a restart causes it to go over all the notifications

**Comment by girokarambole1453** [comment]
![Screenshot_20230831-202043_One UI Home](https://github.com/WFCD/navis/assets/142177308/5dff63c2-ada3-4bbb-b570-eaafbb9079d0)

**Comment by girokarambole1453** [comment]
> Yeah I saw, it's something with the sentient outpost not being there. Since it's stateless a restart causes it to go over all the notifications
> 
> Since I'm at work I can't fix it so I had to just shut it down for the time being

Its compelety aight man. No problem at all. I will close this issue. I dont want to distrupt you at work man. Sorry for that.

**Comment by SlayerOrnstein** [comment]
> > Yeah I saw, it's something with the sentient outpost not being there. Since it's stateless a restart causes it to go over all the notifications
> > 
> > Since I'm at work I can't fix it so I had to just shut it down for the time being
> 
> Its compelety aight man. No problem at all. I will close this issue. I dont want to distrupt you at work man. Sorry for that.

All good it happens, usually I have time for quick stuff but wasn't near a computer so I quickly just shut it down from my phone
